### ISSUE TITLE
Remove dependency on AppCompat

### ISSUE THREAD

**Issue by joharei** [airbnb/Showkase#191]
"Pure" Compose apps have no need for AppCompat, but Showkase depends on it. I don't know if it is actually needed, but it would be nice to remove a big dependency that's probably not used.

**Comment by vinaygaba** [comment]
Yeah don't think there is any reason to still have it. I can get rid of it!
